### DISCUSSION TITLE
Cannot Reach local LAN devices except the router itself.

### DISCUSSION THREAD

**Discussion by pbix** [SmoothWAN/SmoothWAN#142] [General & Help]
I have signed up for tailscale and enabled on my SmoothWAN router.  My remote Windows box connects without problems and I can access the SmoothWAN router remotely.  That's all the good part.  The bad part is that I cannot access any other computer on my home network.

The response to a ping command is "Reply from 100.94.191.127: Destination port unreachable" where 100.94.191.127 is the address of my router I can see at tailscale.com.  I have the connection of my router setup to route the local LAN as documented here [https://smoothwan.com/tail/](url)

My local LAN is 192.168.2.x and if I ping my router at 192.168.2.10 it works fine.  Why would I be able to ping my router using the local LAN IP and not any other device on my network?

The last time I had this working was v0.99.8.  I have now running the latest HF4 release.  Not sure if that is related or not.

**Comment by TalalMash** [comment]
Yes it's in the issues, releasing a new version soon!

**Reply by pbix** [reply]
Thanks,  

I did check the issues before posting and did not see anything.  Could you link the issue for me to read?

**Reply by TalalMash** [reply]
Sorry, I removed that issue by mistake, it's fixed in the latest release.
